### ISSUE TITLE
Turn Feedback On

### ISSUE THREAD

**Issue by LewisMYR** [udacity/frontend-grading-engine#43]
In Quiz 5, Make All the Headers, I don't know how to turn the feedback on when I load my website.  I created the code and see the language.  Please provide solutions.

**Comment by m3d1c** [comment]
I'm having the same problem as well

**Comment by LewisMYR** [comment]
I stopped clicking on the link. I’ve been able to turn on the extension and complete the lessons. I’m not sure why the link doesn’t work.

**Comment by m3d1c** [comment]
Yeah it took me awhile to figure out the I needed to turn on external links
for the extension to read.
